README.md

```diff
@@ -89,6 +90,7 @@ See also [Static Site Generators](#static-site-generators) and  [Content Managem
   * [Known](https://withknown.com/opensource/) - A single website for all your content `Apache` `PHP`
   * [Noddity](http://noddity.com/) - It's a blog, it's a wiki, it's a fast CMS! ([Source code](https://github.com/TehShrike/noddity)) `WTFPL` `Nodejs`
   * [PluXml](http://www.pluxml.org/) - XML-based blog/CMS platform ([Source code](https://github.com/pluxml/PluXml)) `GPL` `PHP`
+  * [WebChao](https://github.com/drazraeltod/webchao) `GPLv3` `Python` `Django`
```

Comment:
There is no description of what this does in the repo.
Please the language tags to one.